include/ibamr/IBFEMethod.h

```diff
@@ -762,6 +844,32 @@ class IBFEMethod : public IBStrategy
      */
     void initializeVelocity(unsigned int part);
 
+    /*!
+     * Get the forward transfer schedule (i.e., from the main hierarchy to the
+     * scratch hierarchy) associated with the given level and index. If
+     * necessary the schedule is created and stored in a map.
+     *
+     * If needed, a SAMRAI::xfer::RefinePatchStrategy object can be provided
+     * for filling ghost data at physical boundaries.
+     */
+    SAMRAI::xfer::RefineSchedule<NDIM>&
+    getForwardScratchSchedule(const int level_number,
+                              const int data_idx,
+                              SAMRAI::xfer::RefinePatchStrategy<NDIM>* rps = nullptr);
+
+    /*!
+     * Get the backward transfer schedule (i.e., from the scratch hierarchy to
+     * the main hierarchy) associated with the given level and index. If
+     * necessary the schedule is created and stored in a map.
+     *
+     * If needed, a SAMRAI::xfer::RefinePatchStrategy object can be provided
+     * for filling ghost data at physical boundaries.
+     */
+    SAMRAI::xfer::RefineSchedule<NDIM>&
+    getBackwardScratchSchedule(const int level_number,
+                               const int data_idx,
+                               SAMRAI::xfer::RefinePatchStrategy<NDIM>* rps = nullptr);
```

Comment:
```suggestion
                               SAMRAI::xfer::RefinePatchStrategy<NDIM>* patch_strategy = nullptr);
```
or `refine_patch_strategy`